zcash_client_backend/src/data_api.rs

```diff
@@ -377,6 +460,9 @@ pub trait InputSource {
     /// The type of errors produced by a wallet backend.
     type Error;
 
+    /// The type used to track unique account identifiers.
+    type AccountId: Copy + Clone + Send + PartialEq + Eq + std::hash::Hash;
```

Comment:
Why does this have the `Copy` constraint whereas on line 531 that constraint does not exist?

Reply:
This addition to the interface also requires a `CHANGELOG` entry.